### PR TITLE
LRDOCS-4890, localized search user guide article

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-4890. We shouldn't publish this until LPS-78119 is published in a Fix Pack.

### CODE REVIEW

File: discover/portal/articles/120-search/07-multilanguage-search.markdown [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+To see an example of site-localized search:
+
+1.  Create a text file named `search-test.txt` with the following contents: 
+
+        Meu time de futebol favorito é o melhor
+
+2. Upload it as a Basic Document to the Documents and Media application.
+
+3.  If your site's language is currently set to English,adding this file will
```
Comment (lipusz) [inline discover/portal/articles/120-search/07-multilanguage-search.markdown]: Missing space after 
>English,adding

File: discover/portal/articles/120-search/07-multilanguage-search.markdown [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+3.  Add a Portuguese (_pt-BR_) translation of the article:
+
+    - Title: _Qual time de futebol é o melhor de todos os tempos?_
+    - Summary: _Existem muitas boas equipes. Qual é o melhor?_
+    - Content: _Aqui estao as 10 melhores equipes do mundo: 1. Selecao
+        brasileira de Futebol 2. O time do Resort Lunar, Waxing Crescent FC..._
+
+4.  Find a search bar widget and enter _time_ as the keyword.
+
+    The first article is returned, and so is the Portuguese translation of the
```
[the remark this answers is not in the document]
Comment (rbohl) [reply discover/portal/articles/120-search/07-multilanguage-search.markdown]: @lipusz I tested on 7.1 CE GA1, and this was the behavior I observed.

**Comment by lipusz** [comment]
Interesting... I must retest it tomorrow on 7.1 DXP GA1

Sent from mobile
--
Project Manager & SME @ Product Care

**Comment by rbohl** [comment]
@lipusz 
I added Categories. I am guessing they work just like Web Content Articles?
Let me know what comes of your testing.

**Comment by lipusz** [comment]
Hi Russ,

dxp-7 1-search-time
This is the result when searching on en_US display locale.

dxp-7 1-search-pt
This is the result when searching on pt_BR display locale.

So I guess you are referring to the 2nd use case in the guide, though it's not written that you need to change the language.

**Comment by rbohl** [comment]
@lipusz  I see now. Looking back at my notes I was assuming the fix for LPS-78119 is already in. I'd rather not publish this until the localized search for web content is working as expected. Make sense? Sorry I misled you above.

**Comment by rbohl** [comment]
@lipusz Do you think this is ready for publication once LPS-78119 makes it into a FP? If so I will send it in for review from Rich, and it will be ready to go once the LPS is backported.

**Comment by lipusz** [comment]
Ready to publish, though, keep in mind the ongoing plans for DM and Multi Language Search:
https://docs.google.com/document/d/1-rWAoh4_bCxLn3EKfV0viLoWIEfLGdHfNl8nbtCMt9c/edit#

https://docs.google.com/document/d/1UtUp8_uMmnWaTH2egD_6BcIpMB_lo6CeZb6kRiGhM9k/edit#

**Comment by rbohl** [comment]
Sent for publication here: https://github.com/sez11a/liferay-docs/pull/3471

**Pull request closed without merging**